Improving the contributing docs (#1114)
pikapkg/snowpack: CONTRIBUTING.md

yarn build:watch
```

## Updating and adding packages

> Since this is a monorepo with several subrepos, if you want to update/add packages in the subrepos like `create-snowpack-app/app-template-11ty` you'll want to run the commands in the target subdirectory like

Reviewer: I never head the temr "subrepos". I think I'd just call it "packages"

Author: probably because I made it up because I couldn't remember "packages" lol

```bash
cd create-snowpack-app/app-template-vue
yarn add vue@latest
```

## Tests

> Our test suite uses a mixture of unit, integration, and snapshot tests. We recommend running these before you submit a PR.

Reviewer: Is "snapshot tests" a thing? I think we can maybe leave out this sentence, or shorten it to just 

```suggestion
We recommend running all tests before you submit a PR
```

### Running tests

> To run tests use:

Reviewer 2: Should this also state to change directory to the top level / root first?

Reviewer: How about 

```suggestion
From the repository's root folder, run
```

Author: True that thanks!

snowpack dev --verbose --reload
```

To test a local version of the `create-snowpack-app` templates use

> NOTE This does not work, trying to figure this one out

Reviewer 3: one thing that could help: splitting this into two different snippets:

To test CLI changes: `node /path/to/snowpack/create-snowpack-app/cli [my-new-dir] --template @snowpack/app-template-vue`

To test a new template: `npx create-snowpack-app [my-new-dir] --template /path/to/template`

Reviewer: good idea 👍🏼 

Let me know if I can help @melissamcewen

Author: I still get `[ERROR] Unable to find "/Users/melissamcewen/sites/snowpack/create-snowpack-app/app-template-vue" in the npm registry.` with either
